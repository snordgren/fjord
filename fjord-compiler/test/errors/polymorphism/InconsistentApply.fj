module errors.parametricPolymorhpism.InconsistentApply

import TestUtil

second : a. a -> a -> a
second x y =
  rm x y

five : Int
five = 
  second "hello" 5
