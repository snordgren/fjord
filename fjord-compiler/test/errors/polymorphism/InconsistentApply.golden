test/errors/polymorphism/InconsistentApply.fj:11:17:
   |
11 |   second "hello" 5
   |                 ^^^
expression has type Int, expected String
